### PR TITLE
Add grep for IIO

### PR DESCRIPTION
**Pull request by tfcollins**
This grep is to get the conversation rolling on merging gr-iio into mainline. I know Most of the focus is on 3.8 right now but just wanted to start the discussion.

### CODE REVIEW

File: grep-0017-iio.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## Abstract
+
+GNU Radio is an invaluable tool for data processing, especially for data sources that can provide large amounts of real-time data like sounds devices and software-defined radios. Such support today is provided by in-tree modules like gr-audio, gr-uhd, gr-comedi, and gr-video-sdl. However, a large category of sensors/DAC/ADC among others supported by the Industrial Input/Output Framework of the linux kernel is not supported in-tree today.
+
+The goal of this GREP is to merge the existing out-of-tree (OOT) module gr-iio into GNU Radio mainline.
+
+## Copyright / License
+
+This GREP is licensed as CC-BY-ND.
+Copyright 2018 The GNU Radio Foundation.
```
Comment (mbr0wn) [inline grep-0017-iio.md]: You can retain copyright for yourself, or maybe for ADI.

**Comment by rgetz** [comment]
:+1: I think this would help alot of people.

**Comment by bhilburn** [comment]
Martin has already been in touch with me about this, and we're all set from
a rights perspective. Queuing Marcus!

Thank you, Travis and Robin! Really excited about this and ADI's continued
support of GNU Radio.

File: grep-0017-iio.md (hunk 1 above)
Comment (mbr0wn) [reply grep-0017-iio.md]: ...and it's 2019 :grin:

**Review by mbr0wn — APPROVED** [review]
I think this is good to go. I'll give it another few days to soak.

File: grep-0017-iio.md (hunk 1 above)
Comment (tfcollins) [reply grep-0017-iio.md]: Updated!

**Comment by smunaut** [comment]
:cry:

**Comment by mbr0wn** [comment]
Merged manually and changed to active. (Also, fixed a bunch of typos :wink: )

**Pull request closed without merging**